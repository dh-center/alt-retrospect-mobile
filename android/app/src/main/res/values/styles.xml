<resources>

    <!-- Base application theme. -->
    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
        <!-- Customize your theme here. -->
        <item name="android:textColor">#000000</item>
        <item name="android:timePickerDialogTheme">@style/DialogDatePicker.Theme</item>
    </style>
    <style name="DialogDatePicker.Theme" parent="Theme.AppCompat.Light.Dialog">
        <item name="colorAccent">#4A75D5</item>
        <item name="android:textColor">#000000</item>
        <item name="android:textColorPrimary">#F3455F</item>
    </style>

</resources>
