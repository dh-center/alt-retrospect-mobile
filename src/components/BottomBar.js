import React, {Component} from 'react';
import {Footer, FooterTab, Button, Icon} from 'native-base';

export default class BottomBar extends Component {
    render() {
        return (
            <Footer>
                <FooterTab>
                    <Button>
                        <Icon name="map" />
                    </Button>
                    <Button>
                        <Icon name="person" />
                    </Button>
                </FooterTab>
            </Footer>
        );
    }
}
