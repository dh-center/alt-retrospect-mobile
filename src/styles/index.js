import * as Alignment from './alignment';
import * as Colors from './colors';
import * as Spacing from './spacing';

export {Alignment, Colors, Spacing};
