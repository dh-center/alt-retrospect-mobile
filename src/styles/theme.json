{
  "color-primary-100": "#FED4CD",
  "color-primary-200": "#FDA09C",
  "color-primary-300": "#F96A72",
  "color-primary-400": "#F3455F",
  "color-primary-500": "#EC0B43",
  "color-primary-600": "#CA084B",
  "color-primary-700": "#A9054E",
  "color-primary-800": "#88034B",
  "color-primary-900": "#710248",
  "color-primary-transparent-100": "rgba(236, 11, 67, 0.08)",
  "color-primary-transparent-200": "rgba(236, 11, 67, 0.16)",
  "color-primary-transparent-300": "rgba(236, 11, 67, 0.24)",
  "color-primary-transparent-400": "rgba(236, 11, 67, 0.32)",
  "color-primary-transparent-500": "rgba(236, 11, 67, 0.4)",
  "color-primary-transparent-600": "rgba(236, 11, 67, 0.48)",
  "color-success-100": "#C8F9CE",
  "color-success-200": "#93F4A8",
  "color-success-300": "#5ADE85",
  "color-success-400": "#30BE6C",
  "color-success-500": "#01944E",
  "color-success-600": "#007F4F",
  "color-success-700": "#006A4D",
  "color-success-800": "#005546",
  "color-success-900": "#004741",
  "color-success-transparent-100": "rgba(1, 148, 78, 0.08)",
  "color-success-transparent-200": "rgba(1, 148, 78, 0.16)",
  "color-success-transparent-300": "rgba(1, 148, 78, 0.24)",
  "color-success-transparent-400": "rgba(1, 148, 78, 0.32)",
  "color-success-transparent-500": "rgba(1, 148, 78, 0.4)",
  "color-success-transparent-600": "rgba(1, 148, 78, 0.48)",
  "color-info-100": "#D0E1FB",
  "color-info-200": "#A2C1F8",
  "color-info-300": "#7099EA",
  "color-info-400": "#4A75D5",
  "color-info-500": "#1946BA",
  "color-info-600": "#12359F",
  "color-info-700": "#0C2785",
  "color-info-800": "#071B6B",
  "color-info-900": "#041259",
  "color-info-transparent-100": "rgba(25, 70, 186, 0.08)",
  "color-info-transparent-200": "rgba(25, 70, 186, 0.16)",
  "color-info-transparent-300": "rgba(25, 70, 186, 0.24)",
  "color-info-transparent-400": "rgba(25, 70, 186, 0.32)",
  "color-info-transparent-500": "rgba(25, 70, 186, 0.4)",
  "color-info-transparent-600": "rgba(25, 70, 186, 0.48)",
  "color-warning-100": "#FFF0CC",
  "color-warning-200": "#FFDC99",
  "color-warning-300": "#FFC466",
  "color-warning-400": "#FFAC3F",
  "color-warning-500": "#FF8400",
  "color-warning-600": "#DB6600",
  "color-warning-700": "#B74D00",
  "color-warning-800": "#933700",
  "color-warning-900": "#7A2700",
  "color-warning-transparent-100": "rgba(255, 132, 0, 0.08)",
  "color-warning-transparent-200": "rgba(255, 132, 0, 0.16)",
  "color-warning-transparent-300": "rgba(255, 132, 0, 0.24)",
  "color-warning-transparent-400": "rgba(255, 132, 0, 0.32)",
  "color-warning-transparent-500": "rgba(255, 132, 0, 0.4)",
  "color-warning-transparent-600": "rgba(255, 132, 0, 0.48)",
  "color-danger-100": "#FDE6D3",
  "color-danger-200": "#FCC7A8",
  "color-danger-300": "#F8A07B",
  "color-danger-400": "#F27B5A",
  "color-danger-500": "#EA4126",
  "color-danger-600": "#C9261B",
  "color-danger-700": "#A81316",
  "color-danger-800": "#870C18",
  "color-danger-900": "#700719",
  "color-danger-transparent-100": "rgba(234, 65, 38, 0.08)",
  "color-danger-transparent-200": "rgba(234, 65, 38, 0.16)",
  "color-danger-transparent-300": "rgba(234, 65, 38, 0.24)",
  "color-danger-transparent-400": "rgba(234, 65, 38, 0.32)",
  "color-danger-transparent-500": "rgba(234, 65, 38, 0.4)",
  "color-danger-transparent-600": "rgba(234, 65, 38, 0.48)"
}