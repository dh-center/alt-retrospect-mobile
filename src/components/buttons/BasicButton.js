import React from 'react';
import {Button, useStyleSheet} from '@ui-kitten/components';
import {styles} from './styles';

export const BasicButton = props => {
    const style = useStyleSheet(styles);
    return (
        <Button
            appearance="ghost"
            status="basic"
            style={style.basic}
            icon={props.renderIcon}
            onPress={props.onPress}
        />
    );
};
