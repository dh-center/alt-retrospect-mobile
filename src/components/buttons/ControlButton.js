import React from 'react';
import {Button} from '@ui-kitten/components';

export const ControlButton = props => {
    return (
        <Button appearance="ghost" status="control" size="giant" {...props} />
    );
};
