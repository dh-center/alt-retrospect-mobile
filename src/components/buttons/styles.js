import {StyleService} from '@ui-kitten/components';

export const styles = StyleService.create({
    control: {
        padding: 0,
    },
});
