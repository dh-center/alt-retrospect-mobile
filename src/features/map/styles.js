import {StyleService} from '@ui-kitten/components';

export const styles = StyleService.create({
    description: {
        borderRadius: 40,
        padding: 20,
        marginBottom: -40,
        height: '50%',
        marginTop: -40,
    },
    topRow: {
        flexDirection: 'row',
        justifyContent: 'space-between',
        alignItems: 'center',
        paddingBottom: 15,
        width: '100%',
    },
    title: {
        maxWidth: '70%',
    },
});
