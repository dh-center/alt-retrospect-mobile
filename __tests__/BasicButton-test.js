/**
 * @format
 */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {light as lightTheme, mapping} from '@eva-design/eva';
import {ApplicationProvider} from '@ui-kitten/components';

import {BasicButton} from '../src/components/buttons/BasicButton';
import {default as appTheme} from '../src/styles/theme';

const theme = {...lightTheme, ...appTheme};

test('renders correctly', () => {
    const tree = renderer
        .create(
            <ApplicationProvider mapping={mapping} theme={theme}>
                <BasicButton />
            </ApplicationProvider>,
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
