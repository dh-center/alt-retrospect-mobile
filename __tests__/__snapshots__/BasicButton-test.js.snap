// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 4,
        "borderWidth": 1,
        "flexDirection": "row",
        "justifyContent": "center",
        "minHeight": 40,
        "minWidth": 40,
        "opacity": 1,
        "paddingHorizontal": 10,
        "paddingVertical": 12,
      }
    }
  />
</View>
`;
